### DISCUSSION TITLE
Custom Type for FormErrors

### DISCUSSION THREAD

**Discussion by WildEgo** [react-hook-form/react-hook-form#8652] [Q&A]
Hey guys, I'm currently returning custom errors and to handle them I need custom types, I was wondering if there's any way of setting them? That or a way to extend the ``message`` field to be cast as whatever field I pass to it.

Regards,
Luis Bizarro.

**Comment by bluebill1049** [comment]
https://react-hook-form.com/api/useform/seterror

**Reply by WildEgo** [reply]
Yes I do set them manually, my main problem is that they're not a string, therefore the type would be wrong
